.travis.yml

```diff
@@ -1,13 +1,19 @@
 language: go
-dist: xenial
+arch:
+  - amd64
+  - arm64
+dist: focal
 go:
-  - 1.12.x
   - 1.13.x
+  - 1.15.x
 script:
   - diff -u <(echo -n) <(gofmt -d *.go)
   - diff -u <(echo -n) <(golint $(go list -e ./...) | grep -v YAMLToJSON)
   - GO111MODULE=on go vet .
   - GO111MODULE=on go test -v -race ./...
+  - if [[ $TRAVIS_CPU_ARCH == "arm64" ]]; then
+      GOARCH=arm GOARM=7 GO111MODULE=on go test -v ./...;
```

Comment:
can you describe what this is doing?

Reply:
I think it is building and testing the 32-bit binary (armv7)

Reply:
[One idea](https://github.com/kubernetes-sigs/yaml/issues/34#issuecomment-726326971) in the original issue was to use the CI setup to run the ARM v7 tests also, as it should be compatible with both v7 and v8.

In case the TravisCI CPU is ARM64, an ARM v7 test is also run, but without `-race`, which is not supported.